### ISSUE TITLE
Importing files stops after a few seconds

### ISSUE THREAD

**Issue by n0bbi** [Radarr/Radarr#9848]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Hello, 

every movie breaks off after a few seconds when importing. Usually about 300 to 600 MB have been copied by then and I see the following error message in the queue: 
"Failed to import movie, Destination already exists."

To fix this, I have to delete the only partially copied file and start the import again. In about 80% of cases, the import stops again with the same message. In about 20% of the cases it then works. 

I cannot find any errors in the log. The only error is that "Destination already exists". It seems to me that it tries to import again during the import and then aborts completely. 

What have I already tried?

- CIFS mount instead of NFS 
- Reboot everything
- Changed categories in Radarr and Downloader

I am currently helping myself by manually copying the file and then importing it with "Manage Files". 

I am at a loss now and don't know where to start. I would appreciate any help and tips in the right direction. Thank you very much! 

### Expected Behavior

Import without canceling. 

### Steps To Reproduce

I need to use the folder mapping because SABnzbd is running on my NAS and Radarr is running on my server. The sharing is done with NFS and there don't seem to be any authorization problems.

![image](https://github.com/Radarr/Radarr/assets/59010376/ca4d7cd1-c409-4bbc-a630-e21cafeec27b)
![image](https://github.com/Radarr/Radarr/assets/59010376/8a2c49a5-bf84-4ba1-8039-886ddef99775)
![image](https://github.com/Radarr/Radarr/assets/59010376/c0612002-4228-4b64-b309-8d32bbe1e060)
![image](https://github.com/Radarr/Radarr/assets/59010376/7fea18bd-0058-48a5-99df-113cd5376aed)
![image](https://github.com/Radarr/Radarr/assets/59010376/9a638c87-965e-4594-8b3d-2df40f4dd1b1)

### Environment

```markdown
- OS: Ubuntu LXC on ProxMox
- Radarr: 5.3.6.8612
- Using Reverse Proxy: traefik 
- Browser: Google Chrome
```

### What branch are you running?

Master

### Trace Logs? **Not Optional**

[trace.txt](https://github.com/Radarr/Radarr/files/14635464/trace.txt)

### Trace Logs have been provided as applicable. Reports will be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by n0bbi** [comment]
I have tried CIFS and NFS and could not find any difference. Here are both fstab entries:

```
#192.168.178.47:/volume1/video   /mnt/video   nfs   defaults     0 0
#192.168.178.47:/volume1/download   /mnt/download   nfs   defaults     0 0
//192.168.178.47/video     /mnt/video        cifs    uid=0,username=xxx,password=xxx,iocharset=utf8,vers=3.0,noperm 0 0
//192.168.178.47/download     /mnt/download        cifs    uid=0,username=xxx,password=xxx,iocharset=utf8,vers=3.0,noperm 0 0
```

```
root@radarr:~# ls -lisa /mnt/video/Movies/Sri\ Asih\ \(2022\)/
total 3236476
2939053       0 drwxr-xr-x 2 root root          0 Mar 18 13:11  .
    261       0 drwxr-xr-x 2 root root          0 Mar 18 13:11  ..
2939054 3236476 -rwxr-xr-x 1 root root 3314151424 Mar 18 13:14 'Sri Asih (2022) Bluray-1080p.mkv'
root@radarr:~# ps aux | grep radarr
root         919  0.0 37.0 3280100 194000 ?      Ssl  13:14   0:06 /opt/Radarr/Radarr -nobrowser -data=/var/lib/radarr/
root        1031  0.0  0.2   3460  1536 pts/3    S+   13:18   0:00 grep --color=auto radarr
root@radarr:~#
```
radarr runs under root and the authorizations are consistently 755

**Comment by n0bbi** [comment]
You are my hero, this error has now cost me several days. I have ruled out that it could be the fileshare, because it is mostly just permissions here. 

My initial tests are very positive. I tried CIFS and NFS, both seem to work now - I hope this is not a coincidence :) 

Thanks for your support! 

I'll close the issue now and open it again if needed.

**Comment by n0bbi** [comment]
Oh no, I was too quick. The error still exists. Unfortunately, it only occurs sporadically, so troubleshooting is difficult.

**Comment by n0bbi** [comment]
Perhaps I have found a solution. I have at least been able to import some files without any problems. 

I can't really explain it, but I have noticed that the web gui is always temporarily unavailable and then the import aborts. In ProxMox I couldn't see a 100% load/RAM Usage, but I increased the CPUs and RAM anyway (previously 1 CPU + 512 MB RAM, now 2 CPU + 1GB RAM). 

Maybe it really was just the environment in the end. I will continue to monitor it.
